### PR TITLE
Cleaning up some old code and DB fields

### PR DESCRIPTION
**Pull request by davidwessman**

### CODE REVIEW

File: db/migrate/20160707064120_remove_elections_posts.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class RemoveElectionsPosts < ActiveRecord::Migration
+  def up
+    drop_table(:elections_posts)
```
Comment (navro) [inline db/migrate/20160707064120_remove_elections_posts.rb]: Remove model, spec, factory and reference in `election` too!

**Comment by navro** [comment]
+1

File: db/migrate/20160707064120_remove_elections_posts.rb (hunk 1 above)
Comment (davidwessman) [inline db/migrate/20160707064120_remove_elections_posts.rb]: This is the old `has_and_belongs_to_many` table, the new one is `election_posts` without plural on elections.

Comment (navro) [inline db/migrate/20160707064120_remove_elections_posts.rb]: > This is the old has_and_belongs_to_many table, the new one is election_posts without plural on elections.

Oh, how confusing :confused:

**Pull request merged**